test/compound/compound.test.ts

```diff
@@ -0,0 +1,127 @@
+import BN from "bn.js";
+import { u8aToHex } from "@polkadot/util";
+import { ApiPromise, WsProvider } from "@polkadot/api";
+import type { KeyringPair } from "@polkadot/keyring/types";
+import { chains } from "@oak-network/config";
+import { OakAdapter } from "@oak-network/adapter";
+import { rpc, types, runtime } from "@oak-network/types";
+import {
+  DEFAULT_TIMEOUT_INITIALIZE,
+  DEFAULT_TIMEOUT_PER_TEST,
+} from "../utils/constants";
+import { findEvent, getKeyringPair } from "../utils/helpFn";
+
+describe("compound", () => {
+  let keyringPair: KeyringPair | undefined;
+  let turingApi: ApiPromise | undefined;
+  let turingAdapter: OakAdapter | undefined;
+  beforeAll(async () => {
+    // Create keyringPair
+    keyringPair = await getKeyringPair();
+    const {
+      DevChains: { turingLocal: turingConfig },
+    } = chains;
+
+    // Initialize adapters
+    turingApi = await ApiPromise.create({
+      provider: new WsProvider(turingConfig.endpoint),
+      rpc,
+      runtime,
+      types,
+    });
+    turingAdapter = new OakAdapter(turingApi, turingConfig);
+    await turingAdapter.initialize();
+  }, DEFAULT_TIMEOUT_INITIALIZE);
+
+  afterAll(async () => {
+    await turingApi?.disconnect();
+  });
+
+  it(
+    "ensure-balance",
```

Comment:
These test case names are so poorly written that they are impossible to read. What does it ensure? Does the wallet’s balance have to be the same as input, or more than that?

Reply:
I have designed this function to return normally if the user's wallet has sufficient balance. Otherwise, it will throw an "Insufficient balance" error.